docker/main/ngen/Dockerfile

```
@@ -664,6 +664,10 @@ RUN if [ "${NGEN_WITH_PYTHON}" == "ON" ]; then \
     fi
 USER ${USER}
 
+ENV VIRTUAL_ENV=/dmod/venv
+RUN python3 -m venv $VIRTUAL_ENV && pip3 install numpy
```

Is it intentional not to activate the `venv` after creating it here? So, should this instead be:
```suggestion
RUN python3 -m venv $VIRTUAL_ENV && source $VIRTUAL_ENV/bin/activate && pip3 install numpy
```

---

Pretty sure it is effectively active if `VIRTUAL_ENV` is set in the environment.

---

I think that is just used for bookkeeping so that the added `deactivate` shell function can remove `$VIRTUAL_ENV` from `$PATH`.